x/stakeibc/keeper/icqcallbacks_validator_exchange_rate.go

```diff
@@ -137,6 +137,9 @@ func ValidatorExchangeRateCallback(k Keeper, ctx sdk.Context, args []byte, query
 
 		EmitFailedLSMLiquidStakeEvent(ctx, hostZone, lsmTokenDeposit, errorMessage)
 		k.Logger(ctx).Error(errorMessage)
+
```

Comment:
piconit: can you clarify the comment on ln 132? "event rejects tx" is unclear to me